Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Cursors/BaseCursor.cs

```diff
@@ -361,16 +361,19 @@ protected virtual void Start()
 
         private void Update()
         {
-            if (!CoreServices.InputSystem.FocusProvider.TryGetFocusDetails(Pointer, out focusDetails))
+            if ((CoreServices.InputSystem != null) && (CoreServices.InputSystem.FocusProvider != null))
             {
-                if (CoreServices.InputSystem.FocusProvider.IsPointerRegistered(Pointer))
+                if (!CoreServices.InputSystem.FocusProvider.TryGetFocusDetails(Pointer, out focusDetails))
                 {
-                    Debug.LogError($"{name}: Unable to get focus details for {pointer.GetType().Name}!");
+                    if (CoreServices.InputSystem.FocusProvider.IsPointerRegistered(Pointer))
+                    {
+                        Debug.LogError($"{name}: Unable to get focus details for {pointer.GetType().Name}!");
+                    }
                 }
-            }
 
-            UpdateCursorState();
-            UpdateCursorTransform();
+                UpdateCursorState();
```

Comment:
Hard to tell from the delta but I think the indentation might be off here.

Reply:
I think the indentation is correct? I added an if statement at the top, because it was blurting out tons of error messages when loading up scenes after switching profiles.

Reply:
I think the indentation is correct? I added an if statement at the top checking if the CoreServices.InputSystem.FocusProvider isn't null, because it was blurting out tons of error messages when loading up scenes after switching profiles.
Related to #6735.